packages/nouns-webapp/src/components/NavLocaleSwitcher/NavLocalSwitcher.module.css

```diff
@@ -0,0 +1,70 @@
+.desktopLanguageButton {
```

Comment:
I don't believe this file is used. If wallet styles are sufficient then generalize those for use here as well.

Reply:
It's used here:
https://sourcegraph.com/github.com/nounsDAO/nouns-monorepo@brianj_add_i18n_support_plus_ja_support/-/blob/packages/nouns-webapp/src/components/NavLocaleSwitcher/index.tsx?L128

I agree though that wallet dropdown styles are used in both places so will rename those to be more general

Reply:
done